packages/millicast-sdk/src/Publish.js

```diff
@@ -19,9 +20,13 @@ const connectOptions = {
   codec: VideoCodec.H264,
   simulcast: false,
   scalabilityMode: null,
-  peerConfig: {}
+  peerConfig: {
+    encodedInsertableStreams: true
+  }
 }
 
+export const DOLBY_SEI_UUID = '6e9cfd2a-5907-49ff-b363-8978a6e8340e'
```

Comment:
Thank you @Santiago-Souto ! Can we add one more please, to make it easier for the end user to compare if the outbound message is a timestamp (open to better name suggestions).

``` javascript
export const SEI_TIMESTAMP_UUID = '9a21f3be-31f0-4b78-b0be-c7f7dbb97250'
```

Reply:
Will use `DOLBY_SEI_DATA_UUID` and `DOLBY_SEI_TIMESTAMP_UUID`, do you agree?